js/wizard/steps/choose-fighter-step/characterProfileMaker.js

```diff
@@ -1,8 +1,5 @@
-import { characterToggler } from './characterToggler.js';
-
-export function characterProfileMaker(index) {
+export function characterProfileMaker(characterDetails) {
```

Comment:
**nitpick:** If it's a maker function I think you should keep it together with other `makers` in `component` dir